docs/installation.md

```diff
@@ -131,26 +131,17 @@ Ingestion tasks populate the database with version control push logs, queued/run
 
 - Then in a new terminal window, run `docker-compose run backend bash`, and follow the steps from the [loading pulse data](pulseload.md) page.
 
-### Ingesting a single push (at a time)
+### Manual ingestion
 
-<!-- prettier-ignore -->
-!!! warning
-    With the end of life of buildbot, this command is no longer able to ingest jobs.
-    For now, after running it, you will need to manually follow the steps from the
-    [loading pulse data](pulseload.md) page.
-
-Alternatively, instead of running a full ingestion task, you can process just
-the jobs associated with any single push generated in the last 4 hours
-([builds-4h]), in a synchronous manner. This is ideal for testing. For example:
+`NOTE`; For all examples of `./manage ingest` you have to use `--root-url` if you need to ingest tasks associated to the [Community Taskcluster instance](https://community-tc.services.mozilla.com).
```

Comment:
Could you make this a bit clearer? You have to include `--root-url https://community-tc.services.mozilla.com` in order to ingest from community-ci, otherwise it will default to firefox-ci.